src/core/bpf/socket_bind/socket-bind-api.bpf.h

```diff
@@ -49,3 +52,4 @@ struct socket_bind_rule {
 };
 
 #define SOCKET_BIND_MAX_RULES 128
+#define SOCKET_BIND_RULE_AF_MATCH_NOTHING 4294967295
```

Comment:
Please write this as `(UINT32_MAX - 1)` instead of a number.

Reply:
UINT32_MAX ?

Reply:
Yes `4294967295` would be `UINT32_MAX`. I've updated the definition